Maxis/Controllers/LoginController.cs

```diff
@@ -12,59 +10,48 @@ namespace Maxis.Controllers
 {
     public class LoginController : Controller
     {
-        
         private readonly IUserService _userService;
-
         public LoginController(IUserService userService)
         {
             _userService = userService;
         }
-        public JsonResult Login(string userName, string password)
+        public JsonResult Login(LoginViewModel loginModel)
         {
-            //var server = WebConfigurationManager.AppSettings["Ldapserver"];
-            //var ldapUser = WebConfigurationManager.AppSettings["Ldapusername"];
-            //var ldapPassword = WebConfigurationManager.AppSettings["Ldappassword"];
-           
-            try
-            {
-                    var roles = Authentication(new LoginViewModel
-                    {
-                        Password = password,
-                        Username = userName
-                    });
-                return Json(roles, JsonRequestBehavior.AllowGet);
-            }
-            catch (Exception ex)
-            {
-                return Json(ex.Message.ToString(), JsonRequestBehavior.AllowGet);
-            }
-           
+            Session["username"] = loginModel.Username;
```

Comment:
do not create session in controller, please use OnAuthorization filter event to validate and set httpcontext.current